python/lsst/pipe/tasks/characterizeImage.py

```diff
@@ -178,70 +178,64 @@ def validate(self):
 
 
 class CharacterizeImageTask(pipeBase.CmdLineTask):
-    """!Measure bright sources and use this to estimate background and PSF of an exposure
+    """Measure bright sources and use this to estimate background and PSF of an exposure
 
-    @anchor CharacterizeImageTask_
+    Notes
+    -----
+    Given an exposure with defects repaired (masked and interpolated over, e.g. as output by IsrTask):
+        - detect and measure bright sources
+        - repair cosmic rays
+        - measure and subtract background
+        - measure PSF
 
-    @section pipe_tasks_characterizeImage_Contents  Contents
+    If you want this task to unpersist inputs or persist outputs, then call
+    the `runDataRef` method (a thin wrapper around the `run` method).
```

Comment:
This is no longer correct; we recently changed all `run` methods to be called `runDataRef` (see [this Community post](https://community.lsst.org/t/api-change-for-tasks-renamed-cmdlinetask-run-rundataref/3100)).